packages/shared/src/components/filters/BlockedWords.tsx

```diff
@@ -113,7 +116,7 @@ export const BlockedWords = (): ReactElement => {
                   });
                 }}
               >
-                upgrade to Plus
+                {plusCta}
```

Comment:
I think we should lowercase the first letter from this variable since the replaced value did have a lowercased first letter which might mean this is a part of a whole sentence.

Reply:
Right, thanks for noticing! Updated ✔️